tests/core_tests/paulis_tests/pauli_test.py

```diff
@@ -1,42 +1,63 @@
 import numpy as np
 import random
-import math
 import pytest
+import math
 from sympleq.core.paulis import PauliSum, PauliString, Pauli
 from sympleq.core.paulis.constants import DEFAULT_QUDIT_DIMENSION
 
-prime_list = [2, 3, 5, 7, 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47]
+# List of prime dimensions to test
+# Choose few for faster testing during development (e.g. until 13)
+# Choose all before PR merging
+prime_list = [2, 3, 5, 7]  # , 11, 13, 17, 19, 23, 29, 31, 37, 41, 43, 47]
+# Number of random tests to perform for each (most) test function
+# Set to a lower number for faster testing during development (e.g. 25)
+# Set to a higher number before PR merging (e.g. 1000)
```

Comment:
I think we could stick with less even, like 50. The CI already takes a while to complete 😅